test-dssspell.sh

```diff
@@ -11,5 +11,5 @@ export DAPP_TEST_NUMBER=$(seth block latest number)
 if [[ -z "$1" ]]; then
   LANG=C.UTF-8 hevm dapp-test --rpc="$ETH_RPC_URL" --json-file=out/dapp.sol.json --dapp-root=. --verbose 1
 else
-  LANG=C.UTF-8 hevm dapp-test --rpc="$ETH_RPC_URL" --json-file=out/dapp.sol.json --dapp-root=. --match "$1" --verbose 1
+  LANG=C.UTF-8 hevm dapp-test --rpc="$ETH_RPC_URL" --json-file=out/dapp.sol.json --dapp-root=. --match "$1" --verbose 3
```

Comment:
Let's keep the verbose in 1, so it doesn't show the full log if everything goes ok.

Reply:
Note that this verbose change only applies when you're matching a test. At least in my workflow, whenever I'm interested in one test, I want to see it with max verbosity.

Maybe if others use `make test match=something` and they want to match many tests, this change doesn't make sense. But otherwise I find it useful.

Reply:
I'm ok with this.

Reply:
If it doesn't show the whole log when not using matching, then I'm ok with it. I wasn't aware that was the case.

Reply:
Yeah sorry, my bad, hadn't seen the `if`.